timur/lib/client/jsx/components/browser/browser.jsx

```diff
@@ -12,6 +12,8 @@
 // Framework libraries.
 import React, {useState, useCallback, useEffect, useMemo} from 'react';
 import {connect} from 'react-redux';
+import 'regenerator-runtime/runtime';
```

Comment:
Should this be in `etna-js/hooks/useAsyncWork` since it appears to set a global?

Reply:
Unfortunately, it can't exist inside of the etna-js package.  For reasons that I haven't determined, the global has to be set in  timur-local package.  I suppose I could put it at the timur/App.js level though.